.gitignore

```diff
@@ -1,6 +1,5 @@
 .DS_Store
 node_modules
-.env
 *.log
```

Comment:
Why did you remove `.env`? This is definitely needed so we don't accidentally upload API credentials